src/core/plugins/spec/selectors.js

```diff
@@ -277,13 +277,14 @@ export function parametersIncludeType(parameters, typeValue="") {
 export function contentTypeValues(state, pathMethod) {
   let op = spec(state).getIn(["paths", ...pathMethod], fromJS({}))
   const parameters = op.get("parameters") || new List()
+
   const requestContentType = (
-      parametersIncludeType(parameters, "file") ? "multipart/form-data"
-    : parametersIncludeIn(parameters, "formData") ? "application/x-www-form-urlencoded"
-    : op.get("consumes_value")
+    op.get("consumes_value") ? op.get("consumes_value")
```

Comment:
I'd prefer if/else blocks over the ternary operators here, Since they were already present, this is OK.